src/content/docs/apm/agents/php-agent/advanced-installation/docker-other-container-environments-install-php-agent.mdx

```diff
@@ -58,12 +62,15 @@ To set up the PHP agent container for Docker:
 
 ## Install agent and daemon in the same container [#install-same-container]
 
+<Callout variant="caution">
+By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. However, for performance reasons, the agent does not wait for those operations to complete before it initializes the connection, which can result in the loss of the first few transactions after a container starts. To prevent this, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to (recommended values)[https://discuss.newrelic.com/t/php-troubleshooting-framework-configuration/119980] of 5s and 15s respectively.
```

Comment:
```suggestion
By default, the first transaction causes the agent to trigger a daemon start and an application connection initialization. For performance reasons, the agent does not wait for those operations to complete before it initializes the connection. This can result in the loss of the first few transactions after a container starts. To prevent this loss, set both `newrelic.daemon.start_timeout` and `newrelic.daemon.app_connect_timeout` entries in the `newrelic.ini` file to our (recommended values)[https://discuss.newrelic.com/t/php-troubleshooting-framework-configuration/119980] of 5s and 15s respectively.
```

Cleaning up a bit of the wording here.